glide-core/src/socket_listener.rs

```diff
@@ -483,6 +483,18 @@ fn get_route(
                 get_slot_addr(&slot_id_route.slot_type)?,
             )),
         ))),
+        Value::ByAddressRoute(by_address_route) => match u16::try_from(by_address_route.port) {
```

Comment:
What if user provides an adress which does not belong to the cluster?

Reply:
then they'll receive an error.